src/js/actions/search/commands.js

```diff
@@ -123,13 +123,11 @@ define(function (require, exports) {
     var _menuCommandSearchOptions = function () {
         var menuStore = this.flux.store("menu"),
             menu = menuStore.getApplicationMenu(),
-            menuMap = menu.rootMap,
             roots = menu.roots.reverse();
 
         var menuCommands = [];
         roots.forEach(function (root) {
-            var currRoot = menuMap.get(root.id),
```

Comment:
Do you know what the purpose of this line was previously? It looks superfluous to me, but I just want to confirm that you agree. (The same functionality is still available via `MenuBar.prototype.byID`, btw.)